### PR TITLE
Erc20 rust test

### PR DESCRIPTION
**Pull request by andrei-marinica**

### CODE REVIEW

File: cmd/test/debugArwen.go [hunk 1]
```diff
@@ -9,6 +9,20 @@ import (
 )
 
 func main() {
+	if len(os.Args) != 2 {
+		panic("One argument expected - the path to the json test.")
+	}
+	jsonTestPath := os.Args[1]
+
+	err := controller.RunSingleIeleTest(
```
Comment (camilbancioiu) [inline cmd/test/debugArwen.go]: Maybe rename `RunSingleIeleTest()` to the generic `RunSingleVMTest()`?

File: cmd/test/executorRun.go [hunk 2]
```diff
@@ -42,12 +43,22 @@ func newArwenTestExecutor() *arwenTestExecutor {
...
 	if len(value) == 0 {
 		return "", nil
 	}
-	fullPath := filepath.Join(testPath, value)
+	var fullPath string
+	if value == "wrc20_arwen.wasm" && te.erc20Path != "" {
+		fullPath = te.erc20Path
+	} else {
+		fullPath = filepath.Join(testPath, value)
+	}
```
Comment (camilbancioiu) [inline cmd/test/executorRun.go]: Why is `value == "wrc20_arwen.wasm"` necessary here? Can't it be treated like any other WASM binary?

File: cmd/test/executorRun.go [hunk 3]
```diff
@@ -42,12 +43,22 @@ func newArwenTestExecutor() *arwenTestExecutor {
 	}
 }
 
+func (te *arwenTestExecutor) setErc20Path(erc20Path string) *arwenTestExecutor {
```
Comment (camilbancioiu) [inline cmd/test/executorRun.go]: Why is there a setter dedicated to the ERC20 test?

File: cmd/test/executorRun.go (hunk 2 above)
Comment (andrei-marinica) [reply cmd/test/executorRun.go]: we have the same tests for 3 different binaries, it was a quick and dirty solution. I'll replace it with something more palatable

File: cmd/test/debugArwen.go (hunk 1 above)
Comment (andrei-marinica) [reply cmd/test/debugArwen.go]: good point, I changed the one with many tests but forgot about this one

**Pull request merged**